core-primitives/types/src/parentchain.rs

```diff
@@ -15,12 +15,19 @@
 
 */
 
-//! Parentchain specific params. Be sure to change them if your node uses different types.
+#[cfg(all(not(feature = "std"), feature = "sgx"))]
```

Comment:
@clangenb 

So now you can atleast pass a `cargo check` outside of the enclave with the latest changes.. Lots of maneuvering to not get cyclic deps.

However.. When doing a `cargo check` inside the enclave it complains that `format!` and std stuff is not found. I forgot how to exactly how to setup the source file with sgx so that std stuff is happy.

Reply:
Except for that this line not necessary, it is correct what you did. However, you also have to activate the `sgx` feature in crates that depend on this crate. Otherwise, it will not work. Regardless, isn't everything you need in core? Hence, you don't need to use sgx at all?